restservice/src/test/java/org/itevents/service/transactional/MyBatisUserServiceTest.java

```diff
@@ -313,4 +315,22 @@ public void shouldThrowServiceExceptionIfOtpIsInvalid() throws Exception {
 
         userService.activateUserWithOtp(stringOtp);
     }
+
+    @Test
+    public void shouldAddLoginInLowerCase() throws Exception {
```

Comment:
@IgorMaksymov `shouldAddSubscriberWithLowerCasedLogin` is more understandable method name